Makefile

```diff
@@ -126,3 +126,14 @@ test_gha:
 
 .PHONY: all
 all: generate test
+
+setup_sanity:
+	poetry install --only sanity --no-root
+
+.PHONY: lint_check
+lint_check: setup_sanity
+	poetry run ruff check .
```

Comment:
Same as above, do `uvx ruff ...` in this file. `uvx` should be available if `uv` is installed. It is just a shorthand for a longer uv command.

Reply:
ok